### ISSUE TITLE
Icons for create text file and create folder/directory missing Hover/Alt text

### ISSUE THREAD

**Issue by clach04** [pldubouilh/gossa-ui#19]
Icons for create text file and create folder/directory missing Hover/Alt text

Also missing for (hidden) manual upload icon
